src/redux/api/dashboardApi.tsx

```diff
@@ -0,0 +1,26 @@
+import { environment } from "@/constants/environment";
+import { IAdminDashboardData } from "@/interfaces/dashboardData";
+import { createApi, fetchBaseQuery } from "@reduxjs/toolkit/dist/query/react";
+import UserTokenService from "../../components/Login/helpers/TokenService";
+
+export const dashboardApi = createApi({
+  reducerPath: "dashboard",
+  baseQuery: fetchBaseQuery({
+    baseUrl: environment.apiBaseUrl,
+    // prepareHeaders: (headers) => {
+    //   const accessToken = UserTokenService.getLocalAccessToken();
+    //   headers.set(
+    //     "authorization",
+    //     accessToken ? `Bearer ${UserTokenService.getLocalAccessToken()}` : ""
+    //   );
+    //   return headers;
+    // },
+  }),
```

Comment:
Why leave these comments here?

Reply:
because the situation at the beginning is that this API can only be accessed by admin, but sometimes the data cannot be get, so I commented it out first, and fixed it after the demo finished.